### DISCUSSION TITLE
ptech-blog/langchain/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#58] [General]
# ptech-blog/langchain/

AI prototyping with LangChain and Streamlit

https://philico-tech.github.io/ptech-blog/langchain/

**Comment by ghost** [comment]
This post was very insightful.

**Comment by ghost** [comment]
test

**Comment by ghost** [comment]
This is a new comment

**Comment by ghost** [comment]
fdsfsdfsd

**Reply by ghost** [reply]
dsadasdsadasdasdasdsa

**Reply by ghost** [reply]
dsadasdasdsa
